enh(mc): automatic commit - Open LDAP (#3113)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-openldap-ldap.md

id: applications-openldap-ldap
title: OpenLDAP
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **OpenLDAP** brings a host template:

* **App-Openldap-LDAP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Openldap-LDAP-custom" label="App-Openldap-LDAP-custom">

| Service Alias | Service Template                      | Service Description                       |
|:--------------|:--------------------------------------|:------------------------------------------|

> | System-Usage  | App-Openldap-System-Usage-LDAP-custom | Check system usage of OpenLDAP using LDAP |

Author: ```suggestion
| System-Usage  | App-Openldap-System-Usage-LDAP-custom | Check system usage of OpenLDAP using the LDAP protocol |
```

> | System-Usage  | App-Openldap-System-Usage-LDAP-custom | Check System Usage of OpenLDAP using LDAP |

Author: ```suggestion
| System-Usage  | App-Openldap-System-Usage-LDAP-custom | Monitor system usage of OpenLDAP using the LDAP protocol |
```

> The services listed above are created automatically when the **App-Openldap-LDAP-custom** host template is used.

</TabItem>
</Tabs>

### Collected metrics & status

Here is the list of services for this connector, detailing all metrics linked to each service.

<Tabs groupId="sync">
<TabItem value="System-Usage" label="System-Usage">

| Metric name                              | Unit  |
|:-----------------------------------------|:------|
| system.connections.current.count         | count |
| system.connections.total.count           | count |
| system.threads.active.percentage         | %     |
| system.traffic.bytespersecond            | B/s   |
| system.operations.search.completed.count | count |
| system.operations.add.completed.count    | count |
| system.operations.bind.completed.count   | count |
| system.operations.unbind.completed.count | count |
| system.operations.delete.completed.count | count |

</TabItem>
</Tabs>

## Prerequisites

The remote server must have an LDAP service running and available. A user and

> password dedicated to monitoring is needed for security reasons.

Reviewer: ```suggestion
password dedicated to monitoring are needed for security reasons.
```
